models/contrib/nish/research.py

```diff
@@ -126,53 +130,51 @@ def run(self, input) -> PoolVolumeInfo:
         else:
             t2_name, t2_symbol, t2_balance = ethereum_token_balance_of_address(token2, pool)
             # Updating coins
-            coin_balances.update({t2_symbol : t2_balance})
+            coin_balances.update({t2_symbol: t2_balance})
             # Updating number of tokens present
             n += 1
             # Updating pool name
-            pool_name = pool_name + '/{}-{}'.format(str(t2_name),str(t2_symbol))
+            pool_name = f'{pool_name}/{t2_name}-{t2_symbol}'

```

Comment:
I'm assuming it isn't on the contract... that's a bummer, uniswap. This would be easy to implement in the contract.